pkg/syncer/volume_health.go

```diff
@@ -116,22 +117,22 @@ func updateVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 	_, foundAnnHealthTS := pvc.Annotations[annVolumeHealthTS]
 	if !found || val != volHealthStatus || !foundAnnHealthTS {
 		// VolumeHealth annotation on pvc is changed, set it to new value
-		log.Debugf("updateVolumeHealthStatus: update volume health annotation for pvc %s/%s from old value %s to new value %s",
+		log.Infof("updateVolumeHealthStatus: update volume health annotation for pvc %s/%s from old value %s to new value %s",
 			pvc.Namespace, pvc.Name, val, volHealthStatus)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealth, volHealthStatus)
 		timeNow := time.Now().Format(time.UnixDate)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealthTS, timeNow)
-		log.Debugf("updateVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s", volHealthStatus, timeNow, pvc.Namespace, pvc.Name)
+		log.Infof("updateVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s", volHealthStatus, timeNow, pvc.Namespace, pvc.Name)
 		_, err := k8sclient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Update(ctx, pvc, metav1.UpdateOptions{})
 		if err != nil {
 			if apierrors.IsConflict(err) {
-				log.Debugf("updateVolumeHealthStatus: Failed to update pvc %s/%s with err:%+v, will retry the update",
+				log.Infof("updateVolumeHealthStatus: Failed to update pvc %s/%s with err:%+v, will retry the update",
 					pvc.Namespace, pvc.Name, err)
 				// pvc get from pvcLister may be stale, try to get updated pvc which bound to pv from API server
 				newPvc, err := k8sclient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(ctx, pvc.Name, metav1.GetOptions{})
 				if err == nil {
 					timeUpdate := time.Now().Format(time.UnixDate)
-					log.Debugf("updateVolumeHealthStatus: updating volume health annotation for pvc %s/%s which "+
+					log.Infof("updateVolumeHealthStatus: updating volume health annotation for pvc %s/%s which "+
```

Comment:
Can we club this with log at line 120 and 125? I'm worried that we are logging the same info multiple times.